### PR TITLE
UICIRC-565: Checked out an item with due date/time in the past

### PR DESCRIPTION
**Pull request by annamelnyk**
## Approach
Fixed due date schedules should start at the beginning of the day (00:00) and end at the end of the day (23:59:59) and the due date should be for the end of the day as well, in the tenant's chosen timezone.

## Refs
https://issues.folio.org/browse/UICIRC-565

## Screenshots
image
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by mkuklis — APPROVED** [review]

File: src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.js [hunk 1]
```diff
@@ -75,12 +80,12 @@ class ScheduleCard extends React.Component {
                   sm={4}
                 >
                   <Field
-                    backendDateStandard="YYYY-MM-DD"
+                    dateFormat="YYYY/MM/DD"
```
Comment (Dmitriy-Litvinenko) [inline src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.js:83]: Could you please move "YYYY/MM/DD" to constant and reuse it across all file?

File: src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.js [hunk 2]
```diff
@@ -130,6 +130,10 @@ class FixedDueDateScheduleForm extends React.Component {
     this.setState({ confirmDelete: true });
   };
 
+  scheduleList = () => {
```
Comment (Dmitriy-Litvinenko) [inline src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.js:133]: Could you please clarify where we use current function?

File: src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.js (hunk 1 above)
Comment (annamelnyk) [reply src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.js]: Done

File: src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.js (hunk 2 above)
Comment (annamelnyk) [reply src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.js]: Nowhere, just forget to clean up. Thanks!

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**